src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -206,6 +206,10 @@ export default {
 			type: Number,
 			required: true,
 		},
+		checkoutButtonEnabled: {
```

Comment:
This prop can probably be removed, since this card isn't being used, and this prop isn't being used.